app/models/explainer.rb

```diff
@@ -0,0 +1,30 @@
+class Explainer < ApplicationRecord
+  include Article
+
+  belongs_to :team
+
+  has_annotations
+
+  before_validation :set_team
+  validates_format_of :url, with: URI.regexp, allow_blank: true, allow_nil: true
+  validates_presence_of :team
+  validate :language_in_allowed_values, unless: proc { |e| e.language.blank? }
+
+  def notify_bots
```

Comment:
Please add a comment to the method body explaining why it was not implemented - or just saying if it was not implemented yet.